tests/wallet/dao_wallet/test_dao_wallets.py

```diff
@@ -581,6 +581,7 @@ async def test_dao_proposals(self_hostname: str, three_wallet_nodes: SimulatorsA
     await asyncio.sleep(1)
     for i in range(1, num_blocks):
         await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(puzzle_hash_0))
+        await asyncio.sleep(2)
```

Comment:
I usually add the sleep after the for loop exits, it doesn't need to sleep after each block. Unless you found this is the only way it passes?